### DISCUSSION TITLE
Is the latest .NET8-rc2 release not yet available as a Docker image?

### DISCUSSION THREAD

**Discussion by SparkyTD** [dotnet/aspnetcore#51691] [Q&A] [answered]
I'm trying to dockerize a project that uses the latest release candidate, but I can't seem to find the correct base image. Currently, I have the following in my Dockerfile:

```
FROM mcr.microsoft.com/dotnet/nightly/sdk:8.0 AS build
```

But the build fails, because the image seems to point to rc1, and my project references some classes (e.g. `RenderModeInteractiveServerAttribute`) that are only available as of rc2. Also, when inspecting the logs of the docker build command, the dotnet tool reports version "8.0.100-rc.1.23463.5", which is incompatible with my workstation SDK (8.0.100-rc.2.23502.2).

Is there a way to use an rc2 image, or do I have to wait until next month for the release?

**Comment by martincostello** [comment, marked as answer]
Looks like you're using the nightly Docker images out of CI, rather than the official release images. Is there a reason you need to use that nightly feed?

There's definitely an RC2 image in the official registry.

**Reply by SparkyTD** [reply]
I have tried the official images, but I'm getting the same errors. Note that it's only complaining about the new render mode classes that were added in rc2. Also, it looks like the .NET version on the official images is also 8.0.100-rc.1.23463.5 (`FROM mcr.microsoft.com/dotnet/sdk:8.0 AS build`). 

I've also tried using sdk:8.0-preview in my Dockerfile, but that points ot an even older version (preview7)

**Reply by SparkyTD** [reply]
Nevermind, I found the rc2 images. I just had to use `mcr.microsoft.com/dotnet/sdk:8.0.100-rc.2` in my Dockerfile. I figured there was a way to pin the rc version, I just wasn't sure where to look for the exact naming of the image.

**Reply by martincostello** [reply]
The "preview" part of the tags was dropped as of RC.1, which is why `sdk:8.0-preview` was pointing at preview 7.
